src/roulette.py

```diff
@@ -869,9 +865,7 @@ def placeBets(self) -> None:
         self.stake -= bet_amount
 
     def playing(self) -> bool:
-        return (
-            self.stake >= self.table.minimum and self.roundsToGo > 0
-        )  # pragma: no cover
+        return super().playing()  # pragma: no cover
```

Comment:
If subclasses don't override, base class inherited method is invoked automatically. pylint also warn this issue

Reply:
Removed.